### DISCUSSION TITLE
Installation of bt-mqtt-gateway addon fails

### DISCUSSION THREAD

**Discussion by Micha70** [FaserF/hassio-addons#198] [General]
Hello, 
I made a new setup for my home assistant with a 64bit image on Raspi4. Everything works now besides the bt-mqtt-gateway. If I try to install the addon I get following message:
`The command '/bin/ash -o pipefail -c apk add --no-cache tzdata bluez bluez-libs sudo bluez-deprecated' returned a non-zero code: 4`

Anyone discovered the same? What is the problem.

**Comment by FaserF** [comment]
Unfortunatly I cannot reproduce it on my x64 pc. And I dont have a 64bit raspi anymore. I will publish a new release soon with an updated base image, maybe that helps
